### ISSUE TITLE
Better integration of audio/video notes plugin with saved favorites

### ISSUE THREAD

**Issue by stanleyguevara** [osmandapp/OsmAnd#14829]
# 🚀 feature request

### Description

Make audio/video notes plugin able to write offline, private photos to favorite places

### Describe the solution you'd like

**Why current solution is bad:**
For example, if I have a favorite place marker already saved and want to add some photos of this place:
- Tap on the favorite > Actions > Take a photo
- The photo marker appears either next to favorite icon (visual noise) or under it (so not visible at first glance)
- Adding multiple photos to the same location seems to create just more markers

Now, after a while I want to check the photos of this place, so:
- I tap on favorite marker
- I get a list of markers (fav place included)
- To see first photo I need to: Tap the undescriptive list entry (no miniature), scroll bottom sheet up, tap on the photo (or tap show)
- To see next photo I need to: Go back, go back again, tap on the fav marker again, read undescriptive list again, remember which photo I've seen previously, tap on the list entry, scroll bottom sheet up, tap on the photo (or tap show).

**That's a lot of friction for a seemingly simple action. How it could be solved:**

Possible solution:
- Tap on the favorite > Actions > Take a photo
- The photo is attached to the fav place marker in a new collapsible row similar to "Online photos" from Mapillary (preferably above it, as it's personal user data = more important = higher in the list)
- All photos stay under one marker and are immediatelly visible by swiping horizontally between them

If the user doesn't want to add favorite place everything works the same, it's just "tap on previous photo note marker" instead of "tap on the favorite". The core idea of this feature request is to integrate a/v notes plugin with existing markers somewhat better, plus improve browsing of said photos.

Excuse my professional ms paint mockup of the solution:
![image](https://user-images.githubusercontent.com/11138100/178477398-4f0471de-df3c-4d3f-a26c-cdd1303a307f.png)

### Describe alternatives you've considered

I've tried audio/video plugin and it's kinda okayish, but it generates a lot of noise, friction and sub-par UX.
The Mapillary is entirely different solution as it's meant to be public, notes are not.

**Comment by vshcherb** [comment]
Yes it was in our plan already, I just couldn't find similar issue. We plan to merge audio / video notes with favorites and make same construction.

**Comment by AnttonC** [comment]
Hi, thanks for this issue that made me discover that audio/video notes were including photos ! 
I was looking everywhere in the app and couldn't find it...

I think "photos" should be added in the title of the "audio/video notes" extension as in my opinion it is way more useful than audio and videos (I would also change the icon).

I totally agree with the issue, can't wait to see offline photos linked to favourites !

**Comment by vshcherb** [comment]
Partially implemented
